source/CoreStartData.cpp

```diff
@@ -49,7 +49,7 @@ void CoreStartData::Save(DataWriter &out) const
 	out.Write("start", identifier);
 	out.BeginChild();
 	{
-		out.Write("system", system->Name());
+		out.Write("system", system->TrueName());
```

Comment:
If I understood @warp-core correctly, he wanted you to continue using `Name()` instead of `TrueName()`?

This might also apply to the variable names, where appropriate; I see some changes to those below as well.

Reply:
warp mentioned the setters, because renaming them from SetName to SetTrueName required a lot of somewhat unrelated changes to other classes. This is not a problem for the getters, and other objects that can have display names (outfits and ships) already use the TrueName / DisplayName convention.